tools/online_editor/src/index.ts

```diff
@@ -351,8 +345,12 @@ import slint_init, * as slint from "@preview/slint_wasm_interpreter.js";
     }
 
     if (component !== undefined) {
-      const instance = component.create(canvas_id);
-      instance.show();
+      if (current_preview_instance === null) {
+        current_preview_instance = component.create("rendering_canvas");
+        current_preview_instance.show();
+      } else {
+        current_preview_instance = component.create_with_existing_window(current_preview_instance);
```

Comment:
The vscode browser preview don't do this.  Should it?

Reply:
Yes, the browser preview should do that, too, IMO. Shall I prepare a patch?

Reply:
Please.

I just tried the preview in the nightly extension on github.dev, and i actually can see all the previous frames behind when moving the mouse (with firefox).
I guess this will fix it.